alpino_query/subtree.py

```diff
@@ -33,6 +33,11 @@ def generate_subtree(twig: etree._Element, remove: Union[str, List[str]]) -> etr
     root = twig.find('node')  # start at 'top' node (leave out alpino_ds node)
     
     subtree = process_twig(root, refpos)
+    if subtree is None:
```

Comment:
Would this always return `//node[@cat="top" and @rel="top"]`?
Maybe it's better to return some constant here or somehow signify this is a (sort of) null query?